### ISSUE TITLE
API endpoint is offline

### ISSUE THREAD

**Issue by knakamura13** [amaldare93/cards-for-humanity#1]
Do you have any plans to bring the API endpoint https://cardsforhumanity.com/ back online?

**Comment by amaldare93** [comment]
Nope. I actually never even started working on this project, I just made a readme and some boilerplate code. I'll delete this repo to avoid further confusion. Sorry
